deckhouse-controller/pkg/controller/module-controllers/source/helpers.go

```diff
@@ -187,8 +187,7 @@ func (r *reconciler) needToEnsureRelease(
 		return false
 	}
 
-	// check the module enabled
-	if !module.ConditionStatus(v1alpha1.ModuleConditionEnabledByModuleConfig) {
+	if module.ConditionUnknown(v1alpha1.ModuleConditionEnabledByModuleConfig) {
```

Comment:
purpose of this replacement?
```suggestion
	if module.ConditionUnknown(v1alpha1.ModuleConditionEnabledByModuleConfig) {
```